guides/customize_tinymce.md

```diff
@@ -73,6 +73,41 @@ That's it. Now the editor can format a block of text as headline 2. Nice!
 Custom buttons always appear in all richtext editors in all elements in Alchemy.
 :::
 
+### Adding editor plugins
+It's possible to configure editor with it's plugins.
+All you have to do is copy (for example) the tinymce list plugin folder to 
+~~~
+vendor/assets/javascripts/tinymce/plugins
+~~~
+
+and add the following content to config/initializers/tinymce.rb
+
+~~~
+Alchemy::Tinymce.init = {
+    plugins: Alchemy::Tinymce.plugins + ['lists'],
+    toolbar: [
+        'bold italic underline | strikethrough subscript superscript | numlist bullist indent outdent | removeformat | alignleft aligncenter alignright | fullscreen',
+        'pastetext charmap hr | undo redo | alchemy_link unlink anchor | code'
+    ]
+}
+~~~
+
+the example above, add the lists plugin to TinyMCE and configure toolbar. The toolbar configuration can be personalized with preferred functions supported by the editor.
```

Comment:
Please change the text to

the example above adds the lists plugin to TinyMCE and configures the toolbar. The toolbar can be personalized with preferred buttons supported by tinymce.